Cargo.toml

```diff
@@ -2,12 +2,23 @@
 name = "mikktspace"
 version = "0.1.0"
 authors = ["Benjamin Wasty <benny.wasty@gmail.com>"]
+description = "Mikkelsen tangent space algorithm"
+documentation = "https://docs.rs/mikktspace"
+repository = "https://github.com/bwasty/mikktspace-rs"
+homepage = "https://github.com/bwasty/mikktspace-rs"
+readme = "README.md"
+keywords = ["3D", "graphics", "algorithm", "tangent"]
+license = "MIT/Apache-2.0"
 build = "build.rs"
+exclude = ["examples/cube.obj", "examples/generate.c"]
```

Comment:
Perhaps even `examples/**/*`?

Reply:
Wouldn't that exclude generate.rs?

Reply:
Yes - I'd only include files necessary for library usage. I don't think many people would go looking for examples in the cryptic path somewhere under `~/.cargo`.
Unrelated: you could now replace `github.com/bwasty` with `github.com/gltf-rs` everywhere.

Reply:
Oh, I understand now. I thought that excluding `generate.rs` would break the example but that's not the case.